ocs_ci/ocs/ui/views.py

```diff
@@ -822,6 +827,28 @@
     "confirm_add_capacity": ('button[data-test-id="confirm-action"]', By.CSS_SELECTOR),
 }
 
+block_pool_4_12 = {
```

Comment:
if block_pool_4_12 and block_pool_4_13 we don't need to repeat them. The point of this dict is to reuse locators. Just pass "block_pool" : block_pool_4_12 for locators["4.13"]

Reply:
Made the changes.